gmplot/gmplot.py

```diff
@@ -454,6 +463,15 @@ def write_ground_overlay(self, f):
             f.write('imageBounds);' + '\n')
             f.write('groundOverlay.setMap(map);' + '\n')
 
+    def write_fitBounds(self, f):
+        if self._fitBounds:
+            f.write('\n')
+            f.write('rectBounds = new google.maps.LatLngBounds(\n')
+            f.write('    new google.maps.LatLng(%f, %f),\n'  % self._fitBounds[:2])
+            f.write('    new google.maps.LatLng(%f, %f));\n' % self._fitBounds[2:])
```

Comment:
I think this is incorrectly flipped; the first value should be the SW point, while the second should be the NE point, as per [Google Maps](https://developers.google.com/maps/documentation/javascript/reference/coordinates#LatLngBounds/).